activerecord/lib/active_record/type/adapter_specific_registry.rb

```diff
@@ -82,7 +82,7 @@ def priority_except_adapter
 
       private
 
-      def matches_adapter?(adapter: nil, **)
+      def matches_adapter?(adapter: nil, **kwargs)
```

Comment:
@carlosantoniodasilva  agreed  but as `**` is good enough !!   It's fixed in `YARD 0.8.7.6`